daffodil-runtime1/src/main/scala/org/apache/daffodil/infoset/XMLTextInfosetInputter.scala

```diff
@@ -191,9 +191,6 @@ object XMLTextInfoset {
 class XMLTextInfosetInputter private (input: Either[java.io.Reader, java.io.InputStream])
```

Comment:
Can this remove the `Either`? Seems like we only need the `InputStream` now?

Reply:
Removing the `Either` seems to cause issue with the `this` constructor. I can try to get that worked out for both JSON and XML infoset inputters if desired.

Reply:
@stevedlawrence To get this to work the `private` keyword need deleted for example for `JsonInfosetInputter` I needed to make the class def be this
```scala
class JsonInfosetInputter(input: java.io.InputStream)
```
instead of having
```scala
class JsonInfosetInputter private (input: java.io.InputStream)
```
would this be how we wanted to do it?

Reply:
Makes sense, we now only need the one constructor so it can be made public.